cmake/modules/SIMDExt.cmake

```diff
@@ -31,18 +31,22 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
   if(HAVE_ARMV8_CRC)
     message(STATUS " aarch64 crc extensions supported")
   endif()
+  CHECK_C_COMPILER_FLAG(-march=armv8-a+crc+crypto HAVE_ARMV8_CRC_CRYPTO_INTRINSICS)
+  if(HAVE_ARMV8_CRC_CRYPTO_INTRINSICS)
+    set(ARMV8_CRC_COMPILE_FLAGS "${ARMV8_CRC_COMPILE_FLAGS} -march=armv8-a+crc+crypto")
```

Comment:
@agraf but @weixiaowilliam is using the GCC intrinsics to implement the CRC digest. i see it's less portable because lower GCC version might not support these intrinsics, but they are easier to write and easier to read.

@weixiaowilliam maybe we can conditionalize this, by defining `CRC32CX` and `CRC32CW` using assembly (i.e. `crc32w` on aach64) if GCC does not support these intrinsics.